global.json

```diff
@@ -55,6 +55,49 @@
         "screenshots": []
       }]
     }, {
+      "id": "ups_project",
+      "priority": 0.410,
+      "name": "Push Hello World",
+      "description": "An example Project using our Unified Push Server",
+      "type": "default",
+      "icon": "icon-refresh",
```

Comment:
Would like to find something more appropriate here in glyphshish, we're already using icon-refresh. inspiration here http://fortawesome.github.io/Font-Awesome/3.2.1/icons/